### PR TITLE
IRIs for team mates

### PR DESCRIPTION
**Pull request by tobiasgrubenmann**

### CODE REVIEW

**Comment by dgarijo** [comment]
@davidlehn I am not sure about if there is a policy about creating individual w3ids on behalf of others. Could someone register all the individuals of their organization?

File: people/ankit-tibrewal/.htaccess [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+#Rewrite rules for Robin Keskisarkka
```
Comment (davidlehn) [inline people/ankit-tibrewal/.htaccess:3]: Looks like a copy & paste mistake for every file?

**Comment by davidlehn** [comment]
@dgarijo No policy yet.  Practically speaking, it would be better to have people register their own ids so it's easier to track who is managing them.  In any case, need some info on who maintainers are.

@tobiasgrubenmann Please add README.md files to the directories listing maintainers of those dirs.  We haven't had people register personal redirects for other people before so I'm unsure what should be listed.  If those other people have github ids, please add those.  Perhaps fallback to name and affiliation.  It's otherwise going to be difficult to know who is allowed to update the rededirects.  Thanks.

**Comment by dgarijo** [comment]
@davidlehn I am not sure about if there is a policy about creating individual w3ids on behalf of others. Could someone register all the individuals of their organization?

**Comment by tobiasgrubenmann** [comment]
I see the problem with creating personal IRIs on behalf of others.
I decided to ask my team mates to create IRIs by themselvers. Hence, I will close this pull request.
Thanks.

**Pull request closed without merging**